bigchaindb/commands/utils.py

```diff
@@ -14,16 +14,46 @@
 from bigchaindb.version import __version__
 
 
-def start_rethinkdb():
+def find_rethinkdb_host():
+    """Utility function to find the bigchainDB host
+    linked by docker with ``--link <container_name>`
+    DISCLAIMER: This is a really dummy function, but should work as long as --link
+    option in docker maintains its consistency
+
+    Returns:
+        IP Address of the host machine plus the default port to connect
+    """
+    default_port = '29015'
+
+    proc = subprocess.Popen(['cat', '/etc/hosts'],
```

Comment:
why not just open the file here? what's the reasoning behind starting a subprocess?